src/app/hooks/useSendTx.ts

```diff
@@ -0,0 +1,118 @@
+import { useCallback, useEffect, useState } from 'react';
+import { TransactionConfig } from 'web3-core';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  selectLoadingTransaction,
+  selectTransactions,
+} from 'store/global/transactions-store/selectors';
+import {
+  Transaction,
+  TxStatus,
+  TxType,
+} from 'store/global/transactions-store/types';
+import { actions } from 'store/global/transactions-store/slice';
+import { useAccount } from './useAccount';
+import { gasLimit } from '../../utils/classifiers';
+import {
+  ResetTxResponseInterface,
+  TransactionOptions,
+} from './useSendContractTx';
+import { walletService } from '@sovryn/react-wallet';
+
+export interface IRawTransactionData {
+  to?: string;
+  data?: string;
+  chainId?: number;
+  value: string;
+  gasPrice: string;
+  gasLimit: string;
+  nonce: number;
+}
+
+export interface SendTxResponseInterface extends ResetTxResponseInterface {
+  send: (config: TransactionConfig, options?: TransactionOptions) => void;
+}
+
+export const useSendTx = (): SendTxResponseInterface => {
+  const transactions = useSelector(selectTransactions);
+  const loading = useSelector(selectLoadingTransaction);
+  const dispatch = useDispatch();
+  const account = useAccount();
+  const [txId, setTxId] = useState<string | null>(TxStatus.NONE);
```

Comment:
```suggestion
  const [txId, setTxId] = useState<string>(TxStatus.NONE);
```

Reply:
Either use my adjustment or leave it the way it was but initialize it with `null`.

Reply:
done